internal/pkg/instrumentation/probe/interfaces.go

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package probe provides instrumentation probe types and definitions.
+package probe
+
+import (
+	"log/slog"
+)
+
+// BaseProbe is the most basic type of Probe. It must support configuration,
+// loading and attaching eBPF programs, running, and closing.
+// All Probes must implement this Base functionality.
+type BaseProbe interface {
+	// ID is a unique identifier for the Probe.
+	// All types of Probes must provide a unique identifier for the framework.
+	ID() ID
+
+	// GetLogger returns an *slog.Logger for this Probe.
+	GetLogger() *slog.Logger
```

Comment:
Isn't this the opposite of what we want in that we want the probes to use the instrumentation logger? When would we need to have the logger from the probe returned?

Reply:
Yeah good point, this is unnecessary